pyproject.toml

```diff
@@ -60,45 +96,19 @@ pytest-postgresql = ">=5"
 ruff = ">=0.0.254"
 types-freezegun = ">=1.1.6"
 types-PyYAML = ">=6.0.7"
+pytest-mock = "^3.14.0"
+mock-gcp = {git = "https://github.com/millin/mock-gcp.git", rev = "0d972df9b6cce164b49f09ec4417a4eb77beb960"}
```

Comment:
question: I skimmed through this, and it seems like upstream is not active. Could you publish your own fork in PyPI? I feel a bit uneasy about including a git source, so if publishing to PyPI is not possible I would like a comment briefly describing what are we getting from the fork that is not in upstream.

Reply:
The difference with upstream is that it mostly doesn't work 😄. I sent a PR to the original author but haven't gotten a response. But I liked the concept and used it as a basis.

I may try publishing this in PyPI (never done it, to be honest).